docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -249,20 +267,44 @@ and maps.
 
 [TIP]
 ==========
-Running our hosted Elasticsearch Service on https://www.elastic.co/cloud[Elastic Cloud]?
-Kibana is enabled automatically in most templates,
-or manually with the {cloud}/ec-enable-kibana.html[flick of a switch].
+Running our hosted {es} Service on https://www.elastic.co/cloud[Elastic Cloud]?
+{kib} is enabled automatically in most templates.
 ==========
 
-We recommend that you install {kib} on the same server as {es},
-but it is not required. If you install the products on different servers, you'll
+We recommend that you install {kib} on the same server as {es}, but it's not
+required. If you install the products on different servers, you'll
 need to change the URL (IP:PORT) of the {es} server in the {kib} configuration
 file, `kibana.yml`, before starting {kib}.
 
 To download and install {kib}, open a terminal window and use the commands that
 work with your system:
 
-*deb, rpm, or linux:*
+* <<deb-rpm-linux,deb>> for Debian/Ubuntu/Redhat/Centos/Fedora
+* <<kibana-mac,mac>> or <<kibana-brew, brew>> for OS X
+* <<kibana-win,win>> for Windows
+
+If this is the first time you're starting {kib}, this command generates a 
+unique link in your terminal to enroll your {kib} instance with {es}.
+
+. In your terminal, click the generated link to open {kib} in your browser.
+
+. In your browser, paste the enrollment token that was generated in the terminal 
+when you started {es}, and then click the button to connect your {kib} instance
+with {es}.
+
+. Log in to {kib} as the `elastic` user with the password that was generated
+when you started {es}.
+
+[NOTE]
+====
+If you need to reset the password for the `elastic` user or other
+built-in users, run the {ref}/reset-password.html[`elasticsearch-reset-password`] tool.
+To generate new enrollment tokens for {kib} or {es} nodes, run the
+{ref}/create-enrollment-token.html[`elasticsearch-create-enrollment-token`] tool.
+These tools are available in the {es} `bin` directory.
+====
+
+[[deb-rpm-linux]]*deb, rpm, or linux:*
```

Comment:
Might consider using the tabbed widgets for platform-specific commands. (I can see why you wouldn't want to go to the trouble, though.)